dev_scripts/env.py

```diff
@@ -262,10 +262,14 @@ def __init__(self, distro, version, runtime):
 
         # Try to autodetect the runtime, if the user has not provided it.
         #
-        # FIXME: Typically Podman runs without sudo, whereas Docker with sudo, but this
-        # is not always the case. We may need to autodetect that as well.
+        # XXX: Typically Podman runs without sudo, whereas Docker runs with sudo only in
+        # Linux environments. Given that we don't tend to use Docker in Linux
+        # environments, we can just ditch the sudo requirement, which breaks
+        # non-interactivity.
+        #
+        # See also: https://github.com/freedomofpress/dangerzone/issues/519
```

Comment:
This feels a bit verbose. I'd be happy as well with just dropping all the comments in this case. But if you feel it's necessary, let's leave it as is.

Reply:
Removed it in 6d4e5cab0172f75f284a9db472b323464c694844.